crud/common/utils.lua

```diff
@@ -606,4 +606,8 @@ function utils.merge_options(opts_a, opts_b)
     return fun.chain(opts_a or {}, opts_b or {}):tomap()
 end
 
+function utils.space_doesnt_exist_msg(space_name)
+    return ("Space %q doesn't exist"):format(space_name)
```

Comment:
Probably we could get this message from:
```
tarantool> box.error.new(box.error.NO_SUCH_SPACE, 'space')
---
- Space 'space' does not exist
...
```

It should be a bit safer if error message will be changed in future. But I won't insist here.

Reply:
To be honest, I had a thought to use separate error class for "Space not found" errors, but decided not to change current behavior. I'm not familiar with `box.error`. Are they supported in 1.10?

Reply:
I think the main problem is that it would be weird to have both `tarantool/errors` and `box.error` at the same time. Maybe the question should get its own ticket after this PR.